pkg/util/metric/mometric/metric.go

```diff
@@ -80,6 +83,7 @@ func InitMetric(ctx context.Context, ieFactory func() ie.InternalExecutor, SV *c
 		moCollector = newMetricCollector(ieFactory, WithFlushInterval(initOpts.exportInterval))
 	}
 	moExporter = newMetricExporter(registry, moCollector, nodeUUID, role)
+	statsLogWriter = newStatsLogWriter(&stats.DefaultRegistry, runtime.ProcessLevelRuntime().Logger().Named("StatsLog"), metric.GetGatherInterval())
 
```

Comment:
@xzxiong @aptend Can I use `runtime.ProcessLevelRuntime().Logger().Named("StatsLog")` here? I have updated the metric_test.go as well, with the relevant change.

Reply:
sure, then the logger_name would be `cn-service.StatsLog` in CN, `dn-service.StatsLog` in DN, `log-service.StatsLog` in LOG.

Reply:
Hi @xzxiong , serviceName is not present inside metric.go. Did you mean, I should add serviceName prefix to `.Named("StatsLog")`? Is it ok to have `Named("StatsLog")` alone?

Reply:
Thanks for confirming.